### ISSUE TITLE
sports.qq.com: ads

### ISSUE THREAD

**Issue by Bonniespace** [uBlockOrigin/uAssets#25951]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://sports.qq.com/kbsweb/game.htm?mid=100000:10022400178`

### Category

ads

### Description

Tencent Sports website could block ads before, but now it can’t

### Other extensions used

IDM

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.0
Chromium: 130
filterset (summary):
 network: 153385
 cosmetic: 59958
 scriptlet: 23320
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 40601-273, now
  ublock-badware: 12137-6, now
  ublock-privacy: 1427-22, now
  ublock-unbreak: 2500-0, now
  ublock-quick-fixes: 212-0, now
  easylist: 77539-616, now
  easyprivacy: 53232-650, now
  urlhaus-1: 21710-0, now
  plowe-0: 3543-0, now
  CHN-0: 25529-57, now
filterset (user): [empty]
trustedset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 4261 ms
 maxAssetCacheWait: 8 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 58
 network:
  qq.com: 57
  miaozhen.com: 1
 extended:
  ##img[data-module="advertisement"]
  ##+js(json-prune, ad)
  ##+js(nowebrtc)
  ##+js(set-constant, CreativePlayerwebPlugin.AD_EVENT.AD_DESTROY,…
  ##+js(set-constant, CreativePlayerwebPlugin.AD_EVENT.AD_LOAD_STA…
  ##+js(json-prune, ads)
```
</details>

**Comment by mapx-** [comment]
test

```adb
||apd-*.*.smtcdns.com/vmind.qqvideo.tc.qq.com/$media,domain=sports.qq.com,redirect=noopmp3-0.1s
```

**Comment by stephenhawk8054** [comment]
Does this work?

```adb
||vm.gtimg.cn^
```

**Comment by mapx-** [comment]
yes, it seems is blocking this player - maybe dedicated to ads ?!

`https://vm.gtimg.cn/thumbplayer`

**Comment by stephenhawk8054** [comment]
Yeah, looks like so.

cc @cjx82630 Sorry, can you check this?

**Comment by cjx82630** [comment]
https://github.com/easylist/easylistchina/commit/4a5cbc0ad1ab59a59355788bebdc4bb593eb7f93
